action.yml

```diff
@@ -77,19 +77,34 @@ inputs:
 outputs:
   checks-failed:
     description: An integer that can be used as a boolean value to indicate if all checks failed.
+    value: ${{ steps.set-checks-failed.outputs.checks-failed }}
 runs:
-  using: "docker"
-  image: "Dockerfile"
-  args:
-    - --style=${{ inputs.style }}
-    - --extensions=${{ inputs.extensions }}
-    - --tidy-checks=${{ inputs.tidy-checks }}
-    - --repo-root=${{ inputs.repo-root }}
-    - --version=${{ inputs.version }}
-    - --verbosity=${{ inputs.verbosity }}
-    - --lines-changed-only=${{ inputs.lines-changed-only }}
-    - --files-changed-only=${{ inputs.files-changed-only }}
-    - --thread-comments=${{ inputs.thread-comments }}
-    - --ignore=${{ inputs.ignore }}
-    - --database=${{ inputs.database }}
-    - --file-annotations=${{ inputs.file-annotations }}
+  using: "composite"
+  steps:
+    - name: Install action dependencies
+      # FIXME: when cpp-linter released to pypi, use that.
+      shell: bash
+      run: python3 -m pip install clang-tools git+https://github.com/cpp-linter/cpp-linter-action@v1
+    - name: Install clang-tools binary executables
+      shell: bash
+      run: clang-tools -i ${{ inputs.version }}
+    - name: Run cpp-linter
+      id: cpp-linter
+      shell: bash
+      run: | 
+        cpp-linter \
+          --style="${{ inputs.style }}" \
+          --extensions=${{ inputs.extensions }} \
+          --tidy-checks="${{ inputs.tidy-checks }}" \
+          --repo-root=${{ inputs.repo-root }} \
+          --version=${{ inputs.version }} \
+          --verbosity=${{ inputs.verbosity }} \
+          --lines-changed-only=${{ inputs.lines-changed-only }} \
+          --files-changed-only=${{ inputs.files-changed-only }} \
+          --thread-comments=${{ inputs.thread-comments }} \
+          --ignore="${{ inputs.ignore }}" \
+          --database=${{ inputs.database }} \
+          --file-annotations=${{ inputs.file-annotations }}
+    - name: set output variable
+      shell: bash
+      run: echo "::set-output name=checks-failed::"
```

Comment:
I tested yesterday and I think it should work.

Reply:
Removing this didn't fix the missing output value.

I'll have to investigate further...